Graphics/GraphicsEngineD3D12/src/SwapChainD3D12Impl.cpp

```diff
@@ -112,7 +112,7 @@ void SwapChainD3D12Impl::InitBuffersAndViews()
         DepthBufferDesc.Usage       = USAGE_DEFAULT;
         DepthBufferDesc.BindFlags   = BIND_DEPTH_STENCIL;
 
-        DepthBufferDesc.ClearValue.Format               = DepthBufferDesc.Format;
+        DepthBufferDesc.ClearValue.Format               = TEX_FORMAT_UNKNOWN;
```

Comment:
I would suggest using `CorrectDXGIFormat(DepthBufferDesc.Format, BIND_DEPTH_STENCIL)` instead

Reply:
Ah, OK.
So the right function here should be 
`GetDefaultTextureViewFormat(DepthBufferDesc.Format, TEXTURE_VIEW_DEPTH_STENCIL, DepthBufferDesc.BindFlags)`